pkg/provider/cty.go

```diff
@@ -0,0 +1,109 @@
+package provider
```

Comment:
FWIW, I wasn't sure what `cty` refers to.

Reply:
I've added a comment clarifying that `go-cty` is the configuration format used by remote state backends in Terraform.